src/app/modules/rup/components/ejecucion/buscador.component.ts

```diff
@@ -261,6 +261,13 @@ export class BuscadorComponent implements OnInit, OnChanges {
         // asignamos el termino de búsqueda para los buscadores de misFrecuentes y sugeridos
         this.search = resultadosSnomed.term;
         if (resultadosSnomed.items.length) {
+
+            resultadosSnomed.items.map((i, pos) => {
+                if (i.term === this.prestacion.solicitud.tipoPrestacion.term) {
```

Comment:
Lo importante siempre es el identificador "conceptId" no el termino. Sabiendo que tiene sinonimos.
```suggestion
                if (i.conceptId === this.prestacion.solicitud.tipoPrestacion.conceptId) {
```